webapp/src/Utils/Scoreboard/Filter.php

```diff
@@ -34,10 +34,18 @@ public function __construct(
     public function getFilteredOn(): string
     {
         $filteredOn = [];
-        if ($this->affiliations) $filteredOn[] = 'affiliations';
-        if ($this->countries)    $filteredOn[] = 'countries';
-        if ($this->categories)   $filteredOn[] = 'categories';
-        if ($this->teams)        $filteredOn[] = 'teams';
+        if ($this->affiliations) {
+            $filteredOn[] = 'affiliations';
```

Comment:
I think this reduces readability and makes it much harder to spot small typos between the different statements.

Reply:
I personally do prefer this, but I'm fine with changing it back.

Reply:
We could even rewrite this for even less typos:
```php
$fields = ['affiliations', 'countries', 'categories', 'teams'];
foreach ($fields as $field) {
  if ($this->$field) {
    $filteredOn[] = $field
  }
}
```